src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -41,11 +41,30 @@ type Props = {
   workflowId: string;
 };
 
+const ContainerWidthBreakpoint = 700;
```

Comment:
is this value something that will be shared amongst other layouts too? should we export it somewhere?

Reply:
I'd hesitate to use this elsewhere. This breakpoint is somewhat specific to the layout of the run workflow button, settings dialog, title, etc. I think on other pages, we'll likely be able to use regular breakpoints, but we're doing something a little more custom here because we are responding to both the state of the page width but also to whether the Drawer is open or not.